doc/unison-manual.tex

```diff
@@ -183,69 +189,16 @@
   Synchronization}, and by University of Pennsylvania's Institute for
 Research in Cognitive Science (IRCS).
 
-\SECTION{Installation}{install}{install}
```

Comment:
Removing this sections breaks a cross-ref in section "Local Usage".

Reply:
fixed I think